Formula/hurl.rb

```diff
@@ -0,0 +1,33 @@
+class Hurl < Formula
+  desc "Run and Test HTTP Requests with plain text and curl"
+  homepage "https://hurl.dev"
+  url "https://github.com/Orange-OpenSource/hurl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "469cca44022d9339fad246fd417b22703258164fd1cdbe040476a1c127791184"
+  license "Apache-2.0"
+  head "https://github.com/Orange-OpenSource/hurl.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  uses_from_macos "curl" => :build
+  uses_from_macos "libxml2" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "packages/hurl")
+    system "cargo", "install", *std_cargo_args(path: "packages/hurlfmt")
+
+    (buildpath/"hurl.1").write `python ci/gen_manpage.py docs/hurl.md`
```

Comment:
```suggestion
    (buildpath/"hurl.1").write Utils.safe_popen_read("python3", "ci/gen_manpage.py", "docs/hurl.md")
```
Same below. We need to use `python3` for this to work on Linux.

Reply:
Thanks @carlocab I've changed and re-pushed an updated formula